windows/cross-compile/build.sh

```diff
@@ -211,30 +223,31 @@ then
   ./Configure $CONFIGURE_OPTIONS
   make
   make install
-  touch $OPENSSL_PREFIX_DIR/done
+  echo -n $OPENSSL_VERSION > $OPENSSL_PREFIX_DIR/done
 
   CONFIGURE_OPTIONS=""
 
   cd ..
   rm -rf ./openssl*
+else
+  echo "Using cached build of OpenSSL `cat $OPENSSL_PREFIX_DIR/done`"
 fi
 
 
 # Qt
 
 QT_PREFIX_DIR="$DEP_DIR/libqt5"
+QT_MAJOR=5
+QT_MINOR=6
+QT_PATCH=3
+QT_VERSION=$QT_MAJOR.$QT_MINOR.$QT_PATCH
 if [ ! -f "$QT_PREFIX_DIR/done" ]
 then
   rm -rf "$QT_PREFIX_DIR"
   mkdir -p "$QT_PREFIX_DIR"
 
   QT_MIRROR=http://qt.mirror.constant.com
 
-  QT_MAJOR=5
-  QT_MINOR=6
-  QT_PATCH=3
-
-  QT_VERSION=$QT_MAJOR.$QT_MINOR.$QT_PATCH
   wget $QT_MIRROR/official_releases/qt/$QT_MAJOR.$QT_MINOR/$QT_VERSION/single/qt-everywhere-opensource-src-$QT_VERSION.tar.xz
```

Comment:
maybe add `--no-verbose` to produce less output?

Reply:
Nah, it's fine as it is, it doesn't hurt anything, not to mention that it's even beneficial to have a verbose output -- pretty sure whoever corss-compiles qTox on a slow internet would want to know the progress.

Reply:
this still displays errors, but hides the progress, which isn't that usefull imo

Reply:
e.g. in this job https://travis-ci.org/qTox/qTox/jobs/288231919 the download creates ~3k lines only for the progress :/

Reply:
This script is not only for Travis, but also for users and it's useful for them to see the progress. 3k for wget lines is a lot, I'm not sure why it does that on Travis, it doesn't print that many lines locally, but removing those ~3k lines won't help with anything, the entire build log is like 20k lines anyway, it doesn't even fit on Travis page, you have to access the log file directly. What would you gain by removing those 3k lines from Travis log and making users unable to track download progress?